website/docs/reference/warehouse-setups/trino-setup.md

```diff
@@ -267,13 +271,13 @@ trino:
 
 <TabItem value="oauth">
 
-To authenticate in dbt-core using OAuth 2 you need only specify: `method: oauth`, in addition to the the above specified host-related fields.
+The only authentication parameter to set for OAuth 2 is `method: oauth`. 
```

Comment:
Can we add a note that users must first enable OAUth within Starburst before being able to use this method?
https://github.com/starburstdata/dbt-trino/issues/275#issuecomment-1489851729